packages/permissions/src/hooks/use-is-authorized/use-is-authorized.ts

```diff
@@ -4,7 +4,7 @@ import {
   hasSomePermissions,
   hasEveryPermissions,
   hasEveryActionRight,
-  hasAppliedDataFence,
+  hasSomeDataFence,
```

Comment:
This is to follow the naming patterns we have already and communicate what this function does: finding if any a match of one data fence.